eslint.config.mjs

```diff
@@ -46,10 +48,13 @@ const MD_LINKS_FOR_DOCS = {
   ...MD_BASE_LINKS
 }
 
+const GLOB_TS = '**/*.?([cm])ts'
+
```

Comment:
Again, this could be `'**/*.{ts,cts,mts}'`, right?

Also, please use this string literal instead of this constant.